packages/PhpDocParser/NodeTraverser/SimpleCallableNodeTraverser.php

```diff
@@ -34,6 +35,7 @@ public function traverseNodesWithCallable(Node | array | null $nodes, callable $
         $nodeTraverser = new NodeTraverser();
         $callableNodeVisitor = new CallableNodeVisitor($callable);
         $nodeTraverser->addVisitor($callableNodeVisitor);
+        $nodeTraverser->addVisitor(new ParentConnectingVisitor());
```

Comment:
this is ensure fill parent Node when replacing Node with other Node via `traverseNodesWithCallable()` 👍